packages/amplify-graphql-api-construct/package.json

```diff
@@ -73,8 +73,8 @@
     "zod"
   ],
   "dependencies": {
-    "@aws-amplify/backend-output-schemas": "^0.2.1",
-    "@aws-amplify/backend-output-storage": "^0.2.0",
+    "@aws-amplify/backend-output-schemas": "^0.4.0",
+    "@aws-amplify/backend-output-storage": "^0.2.2",
     "@aws-amplify/graphql-auth-transformer": "3.2.0",
```

Comment:
Could you make corresponding changes to `packages/amplify-data-construct/package.json` as well? I'd expect us to see the old versions get removed from the `yarn.lock` file.

Reply:
updated